examples/scripts/generate-standalone-files.mjs

```diff
@@ -136,6 +136,10 @@ ${exampleClass.example.toString()}
                     console.warn('Picked WebGPU but example is not supported on WebGPU, defaulting to WebGL2');
                     return 'webgl2';
                 }
+                if (last === 'webgl1' && ${exampleClass.WEBGL1_DISABLED === true}) {
```

Comment:
I'd prefer to avoid 'WEBGL1_DISABLED' flag, as there is no fallback for that device - falling back to webgl2 won't work on device that only support webgl 1.

Other examples that do not support webgl1 simply do not execute code that does not work on webgl 1 .. and then render nothing.

for example

```
if (!device.isWebGL1) {
    // create / use integer textures
}